source/shared/core_sqlsrv.h

```diff
@@ -2377,10 +2377,13 @@ namespace core {
 
     inline void sqlsrv_array_init( _Inout_ sqlsrv_context& ctx, _Out_ zval* new_array TSRMLS_DC) 
     {
-        int zr = ::array_init(new_array);
```

Comment:
Is this necessary? What happens in PHP 7.3 if there is an error with array_init?

Reply:
It used to be a function that returns either a SUCCESS or FAILURE but now a macro in 7.3